app/Models/Dinnerform.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Proto\Models;
+
+use DateTime;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Support\Facades\URL;
+use Vinkla\Hashids\Facades\Hashids;
+
+use Auth;
+
+class Dinnerform extends Model
+{
+    protected $hidden = ['created_at', 'updated_at'];
+
+    /**
+     * The database table used by the model.
+     *
+     * @var string
+     */
+    protected $table = 'dinnerforms';
+
+    public function getPublicId()
+    {
+        return Hashids::connection('dinnerform')->encode($this->id);
+    }
+
+    public static function fromPublicId($public_id)
```

Comment:
Waarom moeten voor dinnerforms the ID's gehasht worden? Die dingen hebben toch geen secret link ofzo? Zou gewoon gaan voor de gewone ID's.